lib/usd/pxrUsdPreviewSurface/usdPreviewSurfaceWriter.cpp

```diff
@@ -45,10 +45,8 @@
 
 PXR_NAMESPACE_OPEN_SCOPE
 
-
-PXRUSDMAYA_REGISTER_WRITER(
-    pxrUsdPreviewSurface,
-    PxrMayaUsdPreviewSurface_Writer);
+PXRUSDMAYA_REGISTER_WRITER(usdPreviewSurface, PxrMayaUsdPreviewSurface_Writer);
+PXRUSDMAYA_REGISTER_WRITER(pxrUsdPreviewSurface, PxrMayaUsdPreviewSurface_Writer);
```

Comment:
The core library shouldn't make any assumptions about the studio plugin. This registration should happen in the plugin.